
From: Paul Jackson <pj@sgi.com>

Document the /proc/<pid>/status fields added in an earlier cpuset patch for
Cpus_allowed and Mems_allowed.

Signed-off-by: Paul Jackson <pj@sgi.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/cpusets.txt |    8 ++++++++
 1 files changed, 8 insertions(+)

diff -puN Documentation/cpusets.txt~cpusets-document-proc-status-allowed-fields Documentation/cpusets.txt
--- 25/Documentation/cpusets.txt~cpusets-document-proc-status-allowed-fields	2004-09-15 20:27:56.369908264 -0700
+++ 25-akpm/Documentation/cpusets.txt	2004-09-15 20:27:56.372907808 -0700
@@ -151,6 +151,14 @@ Each task under /proc has an added file 
 the cpuset name, as the path relative to the root of the cpuset file
 system.
 
+The /proc/<pid>/status file for each task has two added lines,
+displaying the tasks cpus_allowed (on which CPUs it may be scheduled)
+and mems_allowed (on which Memory Nodes it may obtain memory),
+in the format seen in the following example:
+
+  Cpus_allowed:   ffffffff,ffffffff,ffffffff,ffffffff
+  Mems_allowed:   ffffffff,ffffffff
+
 Each cpuset is represented by a directory in the cpuset file system
 containing the following files describing that cpuset:
 
_
